app/components/autocomplete/channel_mention/channel_mention.js

```diff
@@ -181,7 +182,7 @@ export default class ChannelMention extends PureComponent {
             <SectionList
                 keyboardShouldPersistTaps='always'
                 keyExtractor={this.keyExtractor}
-                style={[style.listView, isSearch ? style.search : null]}
+                style={[style.listView, isSearch ? style.search : null, {height: listHeight}]}
```

Comment:
isn't this going to affect the autocomplete in the channel and thread view list height?

Reply:
@enahum I tested `/`, `@`, and `:` on channels and threads on iPhone 6, iPhone X, and Pixel emulators and they all look good.